lib/v2/ikea/cn/utils.js

```diff
@@ -0,0 +1,45 @@
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const generateRandomDeviceId = () => {
+    let id = [10, 6, 6, 6, 14];
+    id = id.map((i) => Math.random().toString(36).substring(2, i));
+    return id.join('-');
+};
+
+const generateRequestHeaders = () => ({
+    'X-Client-Platform': 'WechatMiniprogram',
+    'X-Client-DeviceId': generateRandomDeviceId(),
```

Comment:
**Are you sure** that this id fits the format of what they use?
From `https://www.ikea.cn/cn/zh/offers/family-offers/` the device id is an MD5 like string but **not** a uuid-like string.